.pylintrc

```diff
@@ -329,7 +329,7 @@ indent-after-paren=4
 indent-string='    '
 
 # Maximum number of characters on a single line.
-max-line-length=88
+max-line-length=140
```

Comment:
pre-commit was failing due to other files not changed in this pr due to line length, so I increased it 👀